.changeset/lemon-comics-knock.md

```diff
@@ -0,0 +1,5 @@
+---
+"@rdfjs/types": major
+---
+
+First release, migrated from @types/rdf-js
```

Comment:
One nice thing is that change entries are version controlled and PR-reviewed

Once we're ready for a release, they will be removed and inserted into the `CHANGELOG.md` by CI workflow `release.yml`

Before that happens, that same workflow will maintain a release PR similar to https://github.com/hypermedia-app/Alcaeus/pull/221

Reply:
These files can be added manually, running `npx changeset`, or by virtue of changeset bot's comment, which looks like https://github.com/hypermedia-app/shaperone/pull/126#issuecomment-813006127